sdk/python/test/e2e/utils.py

```diff
@@ -44,3 +64,12 @@ def verify_job_e2e(
     # Print Job logs.
     logging.info(f"\n\n\n{job_kind} logs")
     client.get_job_logs(name, namespace, container=container)
+
+
+def get_pod_spec_scheduler_name(gang_scheduler_name: str) -> str:
+    if gang_scheduler_name == TEST_GANG_SCHEDULER_NAME_SCHEDULER_PLUGINS:
+        return DEFAULT_SCHEDULER_PLUGINS_NAME
+    elif gang_scheduler_name == TEST_GANG_SCHEDULER_NAME_VOLCANO:
+        return ""
```

Comment:
Can you add a comment saying that volcano is not implemented

Reply:
Makes sense.

Reply:
Done.